assignments/objects.js

```diff
@@ -19,26 +19,74 @@ const example = {
 
 // Write your intern objects here:
 
+const intern1 = {
+  'id': 1,
+  'email': 'mmelloy0@psu.edu',
+  name: 'Mitzi',
+  gender: 'F',
+}
```

Comment:
With your objects, try to be consistent in the way they're constructed. It's best practice in general to wrap your object keys in quotes. You are still able to use either dot notation (`intern1.id`) or key notation (`intern1['id']`) so there's no reason not to :+1: